hyperjumptech/monika
# Feat: add cli tests

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Add new tests for monika bats

## How did you implement / how did you fix it  
1.  add print version check
2. add print help check
3. add success starting monika check
4. add print warnings check
5. add able to run probes check
6. add create config from har file check
7. add create config from postman file check
8. add run monika using har file check
9. add run monika using postman collections check

## How to test  
1. run `npm run cli-test`
2. make sure all tests are passed

![Screen Shot 2021-12-23 at 14 58 08](https://user-images.githubusercontent.com/8952493/147207845-571bc5a5-4591-457f-b5db-06c450f71a0a.png)
